bucket/olive-editor-nightly.json

```
@@ -0,0 +1,42 @@
+{
+    "version": "5ef56c80",
+    "description": "Non-linear video editor aiming to provide a fully-featured alternative to high-end professional video editing software.",
+    "homepage": "https://olivevideoeditor.org",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://nightly.link/olive-editor/olive/workflows/ci/master/Olive-5ef56c80-Windows-x86_64.zip.zip",
+            "hash": "f79d3816c1b9a13fa35c17f087d9bcea2b8f29febdc0e74acc5587e8e203b8a8"
+        }
+    },
+    "pre_install": [
+        "7z x \"$dir\\Olive*.zip\" -o\"$dir\\tmp\"",
+        "Move-Item -Path \"$dir\\tmp\\olive-editor\\*\" -Destination \"$dir\\\"",
+        "Remove-Item \"$dir\\tmp\" -Recurse",
```

```suggestion
        "Expand-7zipArchive -Path \"$dir\\Olive*.zip\" -ExtractDir 'olive-editor' -Removal",
```
Can be simplified using the built-in Scoop function.